system/default_ad_api/include/default_ad_api/specs/interface/version.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DEFAULT_AD_API__SPECS__INTERFACE__VERSION_HPP_
+#define DEFAULT_AD_API__SPECS__INTERFACE__VERSION_HPP_
+
+#include "autoware_ad_api_msgs/srv/interface_version.hpp"
+
+namespace ad_api::interface::version
+{
+
+struct T
+{
+  using Service = autoware_ad_api_msgs::srv::InterfaceVersion;
+  static constexpr char name[] = "/api/v0/interface/version";
```

Comment:
Why did you put this under `v0`?
IMO, `/api/version` would be good for this special API.
So I'd like to understand the design concept.

Reply:
One is for consistency with other API. The other is in case the specifications change. Probably the application is fine as long as we keep up to some previous major versions.

However, as you say, this API may have to be treated specially. :thinking:

Reply:
I reconsidered this. Since it is a special case to use different versions of API at the same time, I decided not to support it as the main usage. So remove the version from the API name. I'll provide another API to use the older version.

Reply:
Renamed. https://github.com/autowarefoundation/autoware.universe/pull/704/commits/e9f1d504dc3da5b06caf95da5c62acf71dfc4c1a